src/customRules/trailingCommaInterfaceRule.ts

```diff
@@ -7,6 +7,7 @@ class TrailingCommaInterfaceWalker extends Lint.RuleWalker {
     node.members.filter((propertySignature) => propertySignature.getText().slice(-1) !== ',')
       .forEach((propertySignature) => {
         const lastChar = propertySignature.getText().slice(-1);
+        // tslint:disable-next-line restrict-plus-operands
         const lastCharPosition = propertySignature.getStart() + propertySignature.getWidth() - 1;
```

Comment:
This was getting flagged after updating. I think it might be related to https://github.com/palantir/tslint/issues/2571